modules/stanford_events_importer/stanford_events_importer.module

```diff
@@ -57,6 +57,9 @@ function stanford_events_importer_form_config_pages_stanford_events_importer_for
 
   // Reset all static caches as something is getting stuck.
   drupal_static_reset();
+  // Clear all plugin caches.
+  \Drupal::service('plugin.cache_clearer')
+    ->clearCachedDefinitions();
```

Comment:
Whats this for now? if you wanna clear the migration plugins, theres a better way. https://github.com/SU-SWS/stanford_person/blob/8.x-1.x/modules/stanford_person_importer/stanford_person_importer.module#L169

Reply:
Naw, I need to clear the plugin cache as I need the new config that the form just saved to be overridden by the overrider so the migrate batch can do its thing.

Reply:
Go to https://releasecandidate113-test.sites.stanford.edu/admin/config/importers/events-importer and clear out all the values and save. Then go back and add a new value and hit the save & import button.

Reply:
how about clearing the single config and the migration plugin then? this line and the one above it seem like a heavy hammer approach